src/pages/patterns/cards.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Cards
+description: Cards display a snapshot of information in an easily scannable and digestible way.  
+---
+
+<PageDescription>
+
+Cards display a snapshot of information in an easily scannable and digestible way. Cards encourage user interactions such as tapping, hovering, and clicking to view more details or to take action. 
+They can be used as navigational components or to highlight secondary editorial content.
+
+</PageDescription>
+
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Call-to-action</AnchorLink>
+<AnchorLink>Navigational</AnchorLink>
+<AnchorLink>Featured content</AnchorLink>
+<AnchorLink>Informational</AnchorLink>
+<AnchorLink>Resources</AnchorLink>
+<AnchorLink>Use of icons</AnchorLink>
+<AnchorLink>Related components and patterns</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+Cards are used commonly across digital experiences. They can accommodate a range of content types and are often used to drive or progress users to a supporting page, resource or external destination.
+
+
+## Call-to-action
+
+Cards can be used as a call-to-action (CTA) to direct users to a key piece of content. A CTA card behaves much like a button or text link but is more prominent in size and descriptive in content.
+
+Examples of card components that provide call-to-action functionality:
+
+<Title>Card link</Title>
+
+![Card link](../../images/pattern/cards/cards-cardlink.jpg)
+
+<Caption>
+A single card is placed at the bottom of this section to encourage users to learn more about the preceding content.
+</Caption>
+
+
+### When to use call-to-action cards
+
+* Use to progress a user to related content
+* Use as a replacement for button
+* Use as a replacement for a text link
+
+
+### When not to use call-to-action cards
+
+* Do not use if the desination does not have immediately relevant information
+* Do not use if the next section on the page is a group of cards
+* Do not use if you want the user to stay on the current page 
+
+
+## Navigational
+
+A group of cards can be used to help users navigate to various pieces of information about the subject matter they are interested in learning about or the offering they intend to purchase. 
+
+Examples of card components that serve navigational intent:
+
+<Title>Cards — navigational (coming soon)</Title>
```

Comment:
I wasn't sure what to do here. In some cases on pattern pages, we may want to show designs or visual examples without showing/linking to actual components because either they do not exist _yet_ or they are more helpful "illustrations". 

Maybe I can address this by changing the lead-in text to read: "Examples of card-based designs that serve navigational intent".

Thoughts?

Reply:
I agree with showing visual examples without needing to link to actual coded examples (I think Carbon has done that in some of their patterns??) Some of their examples are wireframes with no content.

I like your suggestion for the lead-in text 👍